classes/01class/exercises/c01-aws01/fabioacp/ANSWER.md

```diff
@@ -0,0 +1,54 @@
+# C01-AWS01
+
+## Commands Execution Output
+
+- Commands to create the first EC2 instance and any additional resource required:
+```bash
+# Create a KeyPair
+aws ec2 create-key-pair --key-name MyKeyPair --query 'KeyMaterial' --output text > MyKeyPair.pem
+
+# Create a EC2 instance with a public IP
+aws ec2 run-instances \
+    --image-id ami-088ff0e3bde7b3fdf \
+    --instance-type t2.micro \
+    --count 1 \
+    --subnet-id subnet-a50917ec \
+    --key-name MyKeyPair \
+    --security-group-ids sg-0983070dc820d22e6 \
+    --associate-public-ip-address
+```
+
+- Commands to connect to the first EC2 instance:
+```bash
+#Set Permission
+CHMOD 400 MyKeyPair.pem
+
+ssh ec2-user@54.206.104.200 -i MyKeyPair.pem 
+```
+
+- Commands to create the second EC2 instance and any additional resource required:
+```bash
+# Create a EC2 instance with a private IP
+aws ec2 run-instances \
+    --image-id ami-088ff0e3bde7b3fdf \
+    --instance-type t2.micro \
+    --count 1 \
+    --subnet-id subnet-a50917ec \
+    --key-name MyKeyPair \
+    --security-group-ids sg-0983070dc820d22e6 \
```

Comment:
if you are using the same security group, I wonder if you are allowing ssh (22) to everywhere. Can you create another sg for the private instance that will only allow the public one to access it?

Reply:
Yes, I used the same SG. I'll create another one.